common/static/coffee/src/discussion/views/discussion_thread_view.coffee

```diff
@@ -22,6 +22,8 @@ if Backbone?
       if @mode not in ["tab", "inline"]
         throw new Error("invalid mode: " + @mode)
 
+      @readOnly = $(".discussion-module").data('read-only')
```

Comment:
The code was relying on returning a falsey value, but I do like making it more explicit. I'll update the other templates to be `data-read-only="false"`.